### ISSUE TITLE
Memory leak in native bullet

### ISSUE THREAD

**Issue by Fadorico** [jMonkeyEngine/jmonkeyengine#489]
See https://hub.jmonkeyengine.org/t/memory-leak-using-multiple-parallel-physics-spaces-native-bullet/35743

Summary:
It seems when using multiple physics spaces in parallel, there is a memory leak in the native bullet code. Test case: 

```
package test;

import com.jme3.app.SimpleApplication;
import com.jme3.bullet.BulletAppState;
import com.jme3.bullet.PhysicsSpace;
import java.util.ArrayList;
import java.util.List;

public class ParallelPhysicsTest extends SimpleApplication{

    private List<PhysicsSpace> physicsSpaces = new ArrayList<>();

    public static void main(String[] args){
        new ParallelPhysicsTest().start();
    }

    @Override
    public void simpleInitApp(){
        createPhysicsSpace();
        createPhysicsSpace();
        createPhysicsSpace();
        createPhysicsSpace();
    }

    public PhysicsSpace createPhysicsSpace(){
        BulletAppState bulletAppState = new BulletAppState();
        bulletAppState.setThreadingType(BulletAppState.ThreadingType.PARALLEL);
        stateManager.attach(bulletAppState);

        physicsSpaces.add(bulletAppState.getPhysicsSpace());

        return bulletAppState.getPhysicsSpace();
    }
}
```

The JVM never throws an out of memory exception, but the OS keeps allocating more memory to the java process at a steady rate. Eventually, the OS will give a low memory warning.

**Comment by stephengold** [comment]
On the forum, @kvainer indicated this bug was still present in JME 3.1. Would someone please check whether it's present in JME 3.2.1 and/or 'master'?

**Comment by stephengold** [comment]
I reproduced the issue with JME 3.1.0-stable, but was unable to reproduce it with 3.2.0-stable or 3.2.1-stable. Apparently the memory leak was fixed between bullet-2.82-r2704 (used in JME 3.1) and bullet3-2.86.1 (used in JME 3.2).
